app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -0,0 +1,57 @@
+ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope', 'networkRouterFormId', 'miqService', function($http, $scope, networkRouterFormId, miqService) {
+  $scope.networkRouterModel = { name: '' };
+  $scope.formId = networkRouterFormId;
+  $scope.afterGet = false;
+  $scope.modelCopy = angular.copy( $scope.networkRouterModel );
+  $scope.model = "networkRouterModel";
+
+  ManageIQ.angular.scope = $scope;
+
+  if (networkRouterFormId == 'new') {
+    $scope.networkRouterModel.name = "";
+    $scope.newRecord = true;
+  } else {
+    miqService.sparkleOn();
+
+    $http.get('/network_router/network_router_form_fields/' + networkRouterFormId).success(function(data) {
+      $scope.afterGet = true;
+      $scope.networkRouterModel.name = data.name;
+
+      $scope.modelCopy = angular.copy( $scope.networkRouterModel );
+      miqService.sparkleOff();
+    });
+  }
+
+  $scope.addClicked = function() {
+    var url = 'create/new' + '?button=add';
+    miqService.miqAjaxButton(url, $scope.networkRouterModel);
+  };
+
+  $scope.cancelClicked = function() {
+    if (networkRouterFormId == 'new') {
+      var url = '/network_router/create/new' + '?button=cancel';
+    } else {
+      var url = '/network_router/update/' + networkRouterFormId + '?button=cancel';
+    }
+    miqService.miqAjaxButton(url, true);
```

Comment:
Still seeing `true` here.. (but given this is `cancel`, you probably just want `miqService.miqAjaxButton(url);` (no data), right?